src/events.ts

```diff
@@ -34,6 +35,10 @@ type AccountProvisionRequestedEvent =  Message & {
   event: 'onAccountProvisionRequested';
 }
 
+type InvestmentAccountProvisionRequestedEvent =  Message & {
+  event: 'onInvestmentAccountProvisionRequested';
```

Comment:
Don't we need the `externalDreamId` field here? Or what is referred to as `investment account uuid` in the ticket: https://getdreams.atlassian.net/browse/AC-2323

Reply:
good point!

Reply:
it's fixed now, let me know if it makes sense